unicorecmsffl/templates/search_box.pt

```diff
@@ -0,0 +1,11 @@
+<div xmlns:metal="http://xml.zope.org/namespaces/metal" 
+	 metal:define-macro="search_box"
+	 xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+     i18n:domain="unicorecmsffl">	
+	<div class="search-box">
+		<form action='/search/' method='GET' class='form-search'>
+		    <input id="q" name='q' type="text" hint='enter search here..' i18n:attributes="hint">
```

Comment:
hint? Ditch this, I think. Fancy version is `placeholder="{an example search term}"`